map_gen/maps/crash_site/commands.lua

```diff
@@ -14,35 +14,67 @@ local Toast = require 'features.gui.toast'
 local Utils = require 'utils.core'
 local DiscordChannelNames = require 'resources.discord_channel_names'
 local set_timeout_in_ticks = Task.set_timeout_in_ticks
+local ScoreTracker = require 'utils.score_tracker'
 
 local Public = {}
 
 function Public.control(config)
 
-local server_player = {name = '<server>', print = print}
+    local server_player = {name = '<server>', print = print}
+    local start_epoch = nil
+    local global_data = {restarting = nil, start_epoch = start_epoch}
```

Comment:
`local start_epoch = nil` isn't used.
```suggestion
    local global_data = {restarting = nil, start_epoch = nil}
```